cardinal/server/server_test.go

```diff
@@ -203,6 +210,95 @@ func TestHandleTransactionWithNoSignatureVerification(t *testing.T) {
 	assert.Equal(t, 1, count)
 }
 
+func TestHandleSwaggerServer(t *testing.T) {
+	count := 0
+	endpoint := "move"
```

Comment:
since this is only used in one place, inline to line 217. 

nit: make it "send-energy".